boa_engine/src/builtins/typed_array/mod.rs

```diff
@@ -2951,7 +2951,7 @@ impl TypedArray {
     }
 
     /// <https://tc39.es/ecma262/#sec-initializetypedarrayfromlist>
-    fn initialize_from_list(
+    pub(crate) fn initialize_from_list(
```

Comment:
Why are these functions only visible to the crate? Why don't we expose them to the developer using boa? If they're in the ECMAScript standard, shouldn't they be exposed and accessible?

Reply:
ECMAScript doesn't define `InitializeTypedArrayFromList` as an exposed API, it's just an abstract operation used internally. The API exposed by the spec are the builtin object properties and the global object properties.